src/backend/db/categories.js

```diff
@@ -8,20 +8,33 @@ import { v4 as uuid } from "uuid";
 export const categories = [
   {
     _id: uuid(),
-    categoryName: "Computer Programming",
+    category: "photography",
     description:
       "Computer programming is the process of designing and building an executable computer program to accomplish a specific computing",
   },
   {
     _id: uuid(),
-    categoryName: "Frontend Development",
+    category: "communication",
     description:
       "Front-end web development, also known as client-side development is the practice of producing HTML, CSS and JavaScript for a website or Web Application",
   },
   {
     _id: uuid(),
-    categoryName: "Backend Development",
+    category: "video editing",
+    description:
```

Comment:
Capitalize the first  letter of the catgeory names,other than that codes looks clean and good to merge.